__mocks__/fs.ts

```diff
@@ -0,0 +1,5 @@
+const fs = jest.genMockFromModule<any>("fs");
+
+fs.existsSync = () => true;
+
+module.exports = fs;
```

Comment:
What are your thoughts on mocking the function instead of mocking the whole module?

```
const mock = jest.spyOn(fs, 'readFileSync');  // spy on fs.readFileSync()
    mock.mockImplementation(() => JSON.stringify({ name: 'myname' }));
```

Would that a better idea? And would it solve the type issue we're currently trying to solve?

Reply:
Unfortunately not.

![Screen Shot 2019-10-31 at 7 29 10 AM](https://user-images.githubusercontent.com/151311/67943354-3a279800-fbb0-11e9-8315-77a3ceecdf2c.png)

Since the instance of the import in the test is different in the actual app code, the mock doesn't apply to the right one AFAIK.

Reply:
Ah, gotcha. Well, I don't think we should hold this back for that reason.

Ping me when you think this is ready to go and I'll approve!